libs/verify-ios-environment.sh

```diff
@@ -18,13 +18,11 @@ fi
 "$(pwd)/libs/verify-ios-ci-environment.sh"
 
 echo ""
-echo "Looks good! Next steps:"
-echo "- Build the XCFramework:"
-echo "    ./megazords/ios-rust/build-xcframework.sh"
-echo ""
-echo " Then you'll be able do one of the following: "
-echo "- Run the tests via the XCode project:"
-echo "    open megazords/ios-rust/MozillaTestServices.xcodeproj"
+echo "Looks good! You can either:"
 echo ""
 echo "- Run the iOS tests via command line:"
 echo "    ./automation/run_ios_tests.sh"
+echo ""
+echo " If you want to just generate the rust binaries"
+echo "- Build the XCFramework:"
+echo "    ./megazords/ios-rust/build-xcframework.sh"
```

Comment:
nit: Should the build script be run before the tests? If so it might make sense to move this addition above the test instructions.

Reply:
The test scripts have builds baked in before running the tests. This is just if the user only needed the binary for some reason. Happy to make this more clear though!